### PR TITLE
chore: adds a TF testing workflow + changes renovate schedule

### PR DESCRIPTION
**Pull request by Gowiem**
## what

- Adds a tf-test workflow for running terraform + tofu tests

## why

- This will ensure we're always running tests on each PR to make sure we don't break anything

## references

- N/A

### CODE REVIEW

File: .github/workflows/tf-test.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            v1-aqua-installer-${{runner.os}}-${{runner.arch}}-
+
+      - name: Install Aqua
+        uses: aquaproj/aqua-installer@4551ec64e21bf0f557c2525135ff0bd2cba40ec7 # v3.0.0
+        with:
+          aqua_version: v2.25.2
+
+      - name: Aqua Install
+        shell: bash
+        run: aqua install --tags terraform
```
Comment (oycyc) [inline .github/workflows/tf-test.yaml:42]: When reading through this, I was wondering if we should use GitHub Action with the matrix strategy to make it more DRY: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#using-a-matrix-strategy

We could replace any usage of `terraform` or `tofu` with something like ${{ matrix.tooling }}. 

But also at the same time, we're just maintaining these two so it doesn't really matter to make it super optimized.

**Review by oycyc — DISMISSED** [review]
This is awesome!! This automation module's potential is so much.

File: .github/workflows/tf-test.yaml (hunk 1 above)
Comment (Gowiem) [reply .github/workflows/tf-test.yaml]: Trying this out!

Comment (Gowiem) [reply .github/workflows/tf-test.yaml]: Working and I like it 👍 We can break it up in the future if terraform + tofu decide to diverge in some way that makes this difficult

Comment (oycyc) [reply .github/workflows/tf-test.yaml]: Yep!

**Review by oycyc — APPROVED** [review]
Matrix is nice

**Pull request merged**